scripts/init.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
-brew install swift-format 
+brew install swift-format
+brew install ktlint release-it
```

Comment:
Do we need `release-it` now?

Reply:
For now i would keep it, then in another PR create whole configuration for releasing all packges